### DISCUSSION TITLE
A toolset to quickly make custom datasets for TTS

### DISCUSSION THREAD

**Discussion by rioharper** [coqui-ai/TTS#2323] [Show and tell]
Hi!

A while back, I made a tutorial for how to create datasets using your own voice and shared it [here](https://github.com/coqui-ai/TTS/discussions/1074) but the last few weeks I have automated that process greatly with the following:

Audio processing to remove non speech elements, overlapping speech, and isolate the same speaker across multiple files
Transcription of audio using OpenAI's whisper, CTC segmentation to clip the audio into usable chunks, and use text normalization to create a voice dataset in the same format as LJSpeech

I have been using it for a few datasets, and while its not perfect its brought down time spent creating custom datasets by 90% or more. 

The github link is https://github.com/rioharper/VocalForge

Let me know what you think!

**Comment by DrewThomasson** [comment]
Amazing! I was looking for something like this! :)
